code/src/components/content/NewTask.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable react/jsx-closing-bracket-location */
+import React, { useState, useRef } from 'react';
+import styled from 'styled-components/macro';
+import { useDispatch, useSelector } from 'react-redux';
+import tasks from 'reducers/tasks';
+import cat from 'reducers/cat';
+import plus from '../../Assets/plus.svg';
+
+const NewTask = ({ bottomRef }) => {
+  const input = useRef(null);
+  const dispatch = useDispatch();
+  const taskListLength = useSelector((store) => store.tasks.tasks).length;
+  const [newTaskText, setNewTaskText] = useState('');
+
+  return (
+    <NewTaskForm
+      onSubmit={(e) => {
+        setTimeout(() => {
+          dispatch(cat.actions.changeCat('surprised'));
+        }, 1);
+        e.preventDefault();
+        dispatch(tasks.actions.addTask(newTaskText));
+        setNewTaskText('');
+        setTimeout(() => {
+          dispatch(cat.actions.changeCat('typing'));
+        }, 1000);
+
+        bottomRef.current.scrollIntoView({
+          block: 'end',
+          behavior: 'smooth',
+          inline: 'end'
+        });
+      }}
+      onFocus={() => {
+        dispatch(cat.actions.changeCatAndPrevious('typing'));
+      }}
+      onBlur={() => {
+        if (taskListLength === 0) {
+          dispatch(cat.actions.changeCat('angel'));
+        } else if (taskListLength > 0 && taskListLength <= 4) {
+          dispatch(cat.actions.changeCat('stressed'));
+        } else if (taskListLength > 4 && taskListLength <= 9) {
+          dispatch(cat.actions.changeCat('dizzy'));
+        } else if (taskListLength > 9) {
+          dispatch(cat.actions.changeCat('dead'));
+        }
+      }}
+    >
+      <NewTaskDiv>
+        <NewTaskButton type="button" onClick={() => input.current.focus()}>
```

Comment:
When i tried the button, i cannot post new task, i can only post task when i press enter.